build.rs

```diff
@@ -29,7 +30,8 @@ fn main() {
         let mut config_include_arg = String::from("-I");
         config_include_arg.push_str(&out_dir);
         config_include_arg.push('/');
-        nasm_rs::compile_library_args("rav1easm", &["src/x86/mc.asm"], &[&config_include_arg, "-Isrc/"]);
+        nasm_rs::compile_library_args("rav1easm", &["src/x86/mc.asm", "src/x86/ipred.asm"], &[&config_include_arg, "-Isrc/"]);
+        println!("cargo:rustc-link-lib=static=rav1easm");
```

Comment:
We should have upstream fix this part sooner or later I guess.

Reply:
What's the deal with the println!, it seems that this is breaking appveyor?

Reply:
It should not and this line is needed in nasm_rs (cf cc-rs)